client/src/js/services/ModalService.js

```diff
@@ -142,11 +143,29 @@ function ModalService(Modal) {
       templateUrl : 'partials/cash/modals/transfer.modal.html',
       controller  : 'CashTransferModalController as CashTransferModalCtrl',
       resolve     : {
-        cashBox:  function cashBoxProvider() { return request.cashbox; }
+        cashBox:  function transferProvider() { return request.cashbox; }
       }
     });
 
     var instance = Modal.open(params);
   }
 
+  /**
+   * Add document Modal
+   */
+  function openAddDocument(request) {
```

Comment:
A better name might be "openUploadDocument", since the modal performs an upload.